docs/tools/piral-inspector.md

```diff
@@ -4,4 +4,203 @@ title: Piral Inspector
 
 # Piral Browser Extension
 
-(tbd)
+## Introduction
+
+To optimally work as a developer with your Piral Instance, and all of its containing Pilets, Piral Inspector offers a great set of capabilities:
+- Getting information about the currently running Piral Instance, e.g. name and version
+- Overview on currently loaded Pilets
+- Adding / Removing of Pilets on-the-fly
+- Overview of registered routes
+- Collection of emitted events
+- Collection of global states
+
+As a developer of your Piral Instance, or your Pilets, you are used to working with a web browser. And this is precisely the best place to get assisted by the Piral Inspector during the development or testing phase.
+
+We provide the Piral Inspector as an extension for the following browsers:
+- Google Chrome (>= v50)
+- Firefox (>= v50)
+- Opera (>= v50)
+
+## Installation of Add-On / Extension
+
+::: summary: Firefox
+
+Type the following URL into the browser address line followed by hitting enter:
+
+about:addons
```

Comment:
Missing backticks (why did you remove them - my suggestion was to just add `plaintext` as the language)

Reply:
Got it.